apps/journeys-admin/src/components/Editor/EditToolbar/Menu/TemplateSettingsDialog/MetadataTabPanel/MetadataTabPanel.tsx

```diff
@@ -34,24 +43,55 @@ export function MetadataTabPanel(): ReactElement {
         onChange={handleChange}
         helperText={t('Publicly visible on template details page')}
       />
-      <FormGroup>
-        <FormControlLabel
-          control={
-            <Checkbox
-              sx={{ mr: 1 }}
-              color="secondary"
-              defaultChecked={values.featured}
-              onChange={handleChange}
-              value={values.featured}
-              name="featured"
-            />
-          }
-          componentsProps={{
-            typography: { color: 'secondary.main', variant: 'subtitle2' }
-          }}
-          label={t('Featured')}
-        />
-      </FormGroup>
+      <LanguageAutocomplete
+        onChange={async (value) => await setFieldValue('language', value)}
+        value={values.language}
+        languages={data?.languages}
+        loading={loading}
+        helperText={t('RTL languages will change the journey flow')}
+      />
+      <Stack direction="row" justifyContent="space-between" alignItems="center">
+        <FormGroup>
+          <FormControlLabel
+            control={
+              <Checkbox
+                sx={{ mr: 1 }}
+                color="secondary"
+                defaultChecked={values.featured}
+                onChange={handleChange}
+                value={values.featured}
+                name="featured"
+              />
+            }
+            componentsProps={{
+              typography: { color: 'secondary.main', variant: 'subtitle2' }
+            }}
+            label={t('Featured')}
+          />
+        </FormGroup>
+        {journey?.publishedAt != null && (
+          <TextField
+            disabled
+            aria-readonly
+            variant="filled"
+            label={format(parseISO(journey?.publishedAt), 'MM/dd/yyyy')}
```

Comment:
use [long localized date](https://date-fns.org/v2.30.0/docs/format):
```suggestion
            label={format(parseISO(journey?.publishedAt), 'P')}
```